evalml/pipelines/utils.py

```diff
@@ -81,6 +82,8 @@ def _get_pipeline_base_class(problem_type):
         return MulticlassClassificationPipeline
     elif problem_type == ProblemTypes.REGRESSION:
         return RegressionPipeline
+    elif problem_type == ProblemTypes.TIME_SERIES_REGRESSION:
```

Comment:
Need to make this change so that the unit tests for make_pipeline don't error out for TIME_SERIES_REGRESSION problem types. 

I'll modify the logic of make_pipeline to handle time series when we integrate time series regression into automl search in #1382